caas/kubernetes/provider/k8s.go

```diff
@@ -303,7 +303,10 @@ func (k *kubernetesClient) APIVersion() (string, error) {
 	if err != nil {
 		return "", errors.Annotatef(err, "got '%s' querying API version", string(body))
 	}
-	return fmt.Sprintf("%s.%s", info.Major, info.Minor), nil
+	version := info.GitVersion
+	// git version is "vX.Y.Z", strip the "v"
+	version = strings.Trim(version, "v")
+	return version, nil
```

Comment:
So this has changed from returning "x.y" to now returning "x.y.z"? There was no test update needed?

Reply:
There's a test that calling APIVersion() invokes the correct ReST URL but to weave in the return result would have required an extra 100s of lines of mocks and/or stubs seeing as the underlying code deals with a rest.Request struct containing an actual embedded http client.